lib/matplotlib/cbook/__init__.py

```diff
@@ -2804,7 +2804,11 @@ def method(self, *args, **kwargs):
                     method.__doc__ = "alias for `{}`".format(prefix + prop)
                     setattr(cls, prefix + alias, method)
         if not exists:
-            raise ValueError("property {} does not exist".format(prop))
+            raise ValueError(
+                "Neither getter nor setter exists for {!r}".format(prop))
 
-    cls._alias_map = alias_d
+    if not hasattr(cls, "_alias_map"):
+        cls._alias_map = {}
```

Comment:
This is dangerous - you need to copy the dict in an `else`, otherwise you'll mutate the one in the base class.

Reply:
good catch, fixed.